src/lib/viewers/image/ImageViewer.js

```diff
@@ -570,6 +572,30 @@ class ImageViewer extends ImageBaseViewer {
         }
     }
 
+    /**
+     * Handler for edit image button click event.
+     *
+     * @private
+     * @return {void}
+     */
+    handleEditImageClick() {
+        this.currentRotationAngle = ((this.currentRotationAngle + 90) % 3600) % 360;
+        this.imageEl.setAttribute('data-rotation-angle', this.currentRotationAngle);
+        this.imageEl.style.transform = `rotate(${this.currentRotationAngle}deg)`;
+        this.emit('rotate');
+
+        // Disallow creating annotations on rotated images
+        if (this.currentRotationAngle === 0) {
+            this.enableAnnotationControls();
+        } else {
+            this.disableAnnotationControls();
+        }
+
+        // Re-adjust image position after rotation
+        this.handleOrientationChange();
```

Comment:
this currently rotates right by 90 degrees, we need to update this to change to the edit image view